files.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 var pythonFiles map[string][]byte
+var rubyFiles map[string][]byte
```

Comment:
does it make sense to just have a "serviceFiles" map and have python and ruby files in one map rather than needing two structures?